### ISSUE TITLE
Put the version of synapse_admin in `manifest.json`

### ISSUE THREAD

**Issue by azmeuk** [Awesome-Technologies/synapse-admin#507]
`manifest.json` supports [version numbers](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/version).
I could use that information, so monitoring tools could check the synapse-admin version installed on my servers and update them when needed.
What do you think about adding the synapse-admin version in the `manifest.json` file?

**Comment by awesome-manuel** [comment]
Good idea. We plan to switch to vite.js and use [vite-plugin-version-mark](https://github.com/ZhongxuYang/vite-plugin-version-mark) to get the correct version from git. Maybe [vite-plugin-pwa]( https://github.com/vite-pwa/vite-plugin-pwa) can be used to generate the `manifest.json`.
